client/src/components/common/SelectMenu.jsx

```diff
@@ -6,25 +6,27 @@ require('../../styles/SelectMenu.scss');
 
 export default function SelectMenu(props) {
   return (
-    <div className="SelectMenu">
+    <div className={`SelectMenu ${props.disabled ? 'disabled' : 'enabled'}`}>
       <Select
         {...props}
-        onChange={option => props.onChange(option.value)}
+        onChange={option => props.onChange(option ? option.value : null)}
         clearable={props.clearable || false}
         searchable={props.searchable || false}
+        resetValue={null}
```

Comment:
Nice catch - this is a relic of a different strategy for resetting the Select when the user clears is that slipped through.